runtime/server/server.go

```diff
@@ -88,17 +89,29 @@ func (s *Server) Ping(ctx context.Context, req *runtimev1.PingRequest) (*runtime
 	return resp, nil
 }
 
+func TimeoutSelector(service, method string) time.Duration {
```

Comment:
Should not be exported (not to be used outside the package). Also please move it below `ServeGRPC`